app/controllers/tmdb_controller.rb

```diff
@@ -89,7 +93,7 @@ def actor_more
         title: movie.title,
         character: credit.character,
         birthday: actor_data.profile.birthday,
-        last_watched_date: movie.most_recent_screening_by(current_user)
+        last_watched_date: movie.max_screening_date
```

Comment:
Since we're selecting `max(screenings.date_watched)` and giving it the alias in the query, we have access to the alias here.